moon/controller_round1.py

```diff
@@ -133,29 +133,37 @@ def on_object_reached(self, data):
         ):
             x,y,z = self.xyz
             print(self.sim_time, "app: Object %s[%d] reached at [%.1f,%.1f], queueing up 19 around" % (object_type, object_index, x, y))
-            heapq.heappush(self.volatile_queue, (1, [object_index, object_type, x, y]))
+            heapq.heappush(self.volatile_queue, (self.sim_time.total_seconds(), [object_index, object_type, x, y]))
```

Comment:
OK, here I see the 19 attempts in priority queue ... but I am not sure if it is OK to completely ignore Z-coordinate? (maybe it is not needed in the report?)

Reply:
Yes, it needs to be a 3d point so that it matches the data types in use but z is ignored for the distance calculation